plugins/catalog-graphql/src/graphql/module.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Logger } from 'winston';
+import fs from 'fs';
+import path from 'path';
+import { GraphQLModule } from '@graphql-modules/core';
+import { Resolvers, CatalogEntity, CatalogQuery } from './types';
+import { Entity } from '@backstage/catalog-model';
+import { Config } from '@backstage/config';
+import { CatalogClient } from '../service/client';
+
+export interface ModuleOptions {
+  logger: Logger;
+  config: Config;
+}
+
+const parseToCatalogEntities = (e: Entity): CatalogEntity => ({
+  ...e,
+  metadata: {
+    ...e.metadata,
+    annotations: Object.entries(
+      e.metadata.annotations ?? {},
+    ).map(([key, value]) => ({ key, value })),
+  },
+});
+
+export async function createModule(
+  options: ModuleOptions,
+): Promise<GraphQLModule> {
+  const typeDefs = await fs.promises.readFile(
+    path.resolve(__dirname, '..', 'schema.gql'),
```

Comment:
does the schema file follow along properly to the dist folder during build?

Reply:
Hmm - maybe it doesn't, @Rugvip any suggestion here as a nice way to include the schema in the built asset?

Reply:
https://github.com/spotify/backstage/commit/650830cf4f54db4609a0145b67b0c056633e27e0#diff-8e8630f7f7a6bc56707c386ce224a380 should help :grin:

In general I'm considering whether we should standardize on an `assets/` dir where we put all static assets that we wanna ship with backend packages. And then likely also include an API for at least resolving relative to the assets dir.